lib/matplotlib/tests/test_axes.py

```diff
@@ -785,6 +785,13 @@ def test_hexbin_log():
     plt.colorbar(h)
 
 
+def test_hexbin_log_clim():
+    x, y = (np.arange(200) / 200).reshape((2, 100))
```

Comment:
```suggestion
    x, y = np.arange(200) / 200).reshape((2, 100)
```

Reply:
I think this is mistaken, since the suggestion leaves unmatched parentheses. Though this line can be simplified to just `np.arange(200).reshape((2, 100))`, which I'll do.

Reply:
Ah, you're right, I misread it, but that works too.